d2l-button-toggle.js

```diff
@@ -1,153 +1,128 @@
-				@apply --d2l-button;
-				@apply --d2l-button-shared;
-				@apply --d2l-label-text;
-				border-radius: 0;
-				border-width: 1px 1px 1px 0px;
+			button {
 				font-family: inherit;
 				padding: 0.5rem 1.5rem;
+				@apply --d2l-button-shared;
+				@apply --d2l-label-text;
+				@apply --d2l-button;
+				@apply --d2l-button-clear-focus;
+				width: 100%;
+				border-radius: var(--d2l-button-toggle-border-radius);
+				border-width: var(--d2l-button-toggle-border-width);
 			}
-			.d2l-button-toggle-button {
+			/* Firefox includes a hidden border which messes up button dimensions */
+			button::-moz-focus-inner {
+				border: 0;
+			}
+			button,
+			button[disabled]:hover,
+			button[disabled]:focus,
+			:host([active]) button[disabled] {
 				background-color: var(--d2l-color-regolith);
+				border-color: var(--d2l-color-mica);
+				color: var(--d2l-color-ferrite);
 			}
-			:host(:not([disabled])) .d2l-button-toggle-button:hover,
-			:host(:not([disabled])) .d2l-button-toggle-button:focus,
-			.d2l-button-toggle-button[aria-pressed]
-			{
+
+			button:hover,
+			button[aria-pressed="true"], 
+			:host([active]) button,
+			:host(.d2l-button-hover) button {
 				background-color: var(--d2l-color-gypsum);
 			}
-			.d2l-button-toggle-button:focus,
-			.d2l-button-toggle-button[aria-pressed]:focus {
-				box-shadow: 0 0 1px 0 var(--d2l-color-sylvite) inset, 0 0 3px 1px #99C5E5;
-				position: relative;
+
+			button:hover,
+			button[aria-pressed="true"], 
+			:host(.d2l-button-hover) button, 
+			:host([active]) button {
+				@apply --d2l-button-hover;
 			}
-			.d2l-button-toggle-container {
-				list-style: none;
-				white-space: no-wrap;
-				display: inline-block;
+			button:focus, :host(.d2l-button-focus) button {
+				@apply --d2l-button-focus-plus-border;
+				position: relative;
 			}
-			.d2l-button-toggle-container li {
-				display: inline-block;
+
+			button[disabled] {
+				opacity: 0.5;
+				cursor: default;
 			}
 		</style>
-		<d2l-offscreen><h2 id="d2l-button-toggle-title">[[title]]</h2></d2l-offscreen>
-		<ul class="d2l-button-toggle-container" role="toolbar" aria-labelledby="d2l-button-toggle-title">
-			<dom-repeat items="[[buttons]]" as="b">
-				<template><li role="presentation">
-					<button 
-						class="d2l-button-toggle-button" 
-						id$="[[_getButtonId(b.key)]]" 
-						on-click="_selectionChanged"
-						aria-pressed$="[[_shouldBeDefaultSelected(index)]]"
-						disabled="[[disabled]]"
-					>[[b.text]]</button>
-				</li></template>
-			</dom-repeat>
-		</ul>
+		<button 
+			aria-expanded$="[[ariaExpanded]]"
+			aria-haspopup$="[[ariaHaspopup]]"
+			aria-label$="[[label]]"
+			aria-pressed$="[[_pressedString]]"
+			class="d2l-focusable"
+			disabled$="[[disabled]]"
+			autofocus$="[[autofocus]]"
+			form$="[[form]]"
+			formaction$="[[formaction]]"
+			formenctype$="[[formenctype]]"
+			formmethod$="[[formmethod]]"
+			formnovalidate$="[[formnovalidate]]"
+			formtarget$="[[formtarget]]"
```

Comment:
Most of these attributes don't have corresponding properties (and shouldn't) -- you only need to specify the ones you actually want to set.